sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/ExceptionNotThrown.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Tests.Diagnostics
+{
+    class ExceptionNotThrown
+    {
+        public void Test1()

```

Comment:
A bit more descriptive test names would be more useful. Not a big deal in this case.